modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -347,19 +294,19 @@ public void getInfo(PrintWriter pw) {
     public void messageArrived(CellMessage message) {}
 
     private String getNiceDayOfCalendar(Calendar calendar) {
-        return _dayOfCalendar.get().format(calendar.getTime());
+        return calendar.getTime().toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime().format(_dayOfCalendar);
```

Comment:
i thought that is easier converting Calendar object to LocalDateTime object then call the format method